scripts/install.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/sh
+#
+# This script installs the latest release of Tenzir.
+#
+
+# Treat unset variables as an error when substituting.
+set -u
+
+# Abort on error.
+set -e
```

Comment:
If you can use bash rather than sh, it would be helpful to have `set -euo pipefail` here. That usually helps a lot with checking correctness.

Reply:
We're not using Bash on purpose.

Reply:
Why is that? `/usr/bin/env bash` is available basically everywhere.

Reply:
There's no need in using Bash just for `pipefail`. POSIX Shell makes the script far more accessible.

Reply:
Resolving this for now. We can reconsider Bash if we need more advanced scripting fu.